nnpackage/schema/circle_schema.fbs

```diff
@@ -76,9 +76,15 @@ table CustomQuantization {
   custom:[ubyte] (force_align: 16);
 }
 
+// Block quantization: from ggml quantization (https://github.com/ggerganov/ggml)
+table CircleBlockQuantization {
+  name:string;
+}
+
 // Represents a specific quantization technique's parameters.
 union QuantizationDetails {
   CustomQuantization,
+  CircleBlockQuantization
```

Comment:
Is the prefix `Circle` necessary to avoid name conflict from flatbuffers generated files? I guess `GGMLBlockQuantization` may be better. It makes it clear what it means.